Server/scripts/fetchYearMaps.ts

```diff
@@ -210,7 +218,15 @@ const getBNsApiCallRawData = async (beatmapSetID: BeatmapsetID): Promise<null |
 }
 
 async function insertBeatmap (apiBeatmap: APIBeatmap) {
-    let beatmap = new Beatmap;
+    let beatmap = await Beatmap.findOne({
+        where: {
+            ID: apiBeatmap.id,
+        },
+        relations: ["beatmapset"],
```

Comment:
Relation is unneeded here, it is already in-memory or fetched by the `getBeatmapSet` call further down